data/AKHIL-GIREESH.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "AKHIL GIREESH",
+  "type": "personal",
+  "bio": "19 | Incipient Web 3 Developer | Blockchain & AI Enthusiast | Amateur Surrealism Artist",
+  "links": [
+    {
+      "name": "Reach out to me on Twitter",
+      "url": "https://twitter.com/_0x416B68696C_",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Reach out to me on Github",
+      "url": "https://twitter.com/_0x416B68696C_",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Reach out to me on LinkedIn",
+      "url": "https://twitter.com/_0x416B68696C_",
```

Comment:
Please kindly add your LInkedin link here (not twitter)

Reply:
Fixed! Thank you for pointing it out